doc/python_api/rst/bge_types/bge.types.KX_FontObject.rst

```diff
@@ -50,3 +50,10 @@ base class --- :class:`KX_GameObject`
 
       :type: float (0.0001 to 40.0)
 
+   .. method:: getTextSize()
```

Comment:
I prefer use an attribute instead of function: `font.textSize`. Also we already have an attribute named `size`, maybe you should use something like:
```
font.dimension
font.textDimension
```
If `font.text…`is not redundant.